data/MutiatBash.json

```
@@ -0,0 +1,20 @@
+{
+  "name":"Mutiat Bashua",
+  "bio":"Frontend Developer | Mobile App Developer | [Github Profile](https://github.com/MutiatBash)",
+  "links":[
+    {
+  "group":"Socials",
+  "name":"Twitter:",
+  "url":"https://twitter.com/dev_mutiah",
+  "icon":"FaTwitter",
+  "color":"#00ACEE",
```

```suggestion
  "color":"#00ACEE"
```
The trailing comma isnot required